### PR TITLE
Adding book link for Practical MongoDB Aggregations

### PR DESCRIPTION
**Pull request by pkdone**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing. F
  - [Builder Book](https://builderbook.org) - Learn how to build a full stack JavaScript web app from scratch
  - [MongoDB Applied Design Patterns (Rick Copeland)](https://www.oreilly.com/library/view/mongodb-applied-design/9781449340056/)
  - [The Little MongoDB Book](https://www.openmymind.net/2011/3/28/The-Little-MongoDB-Book/) - Basic introduction
+ - [Practical MongoDB Aggregations](https://www.practical-mongodb-aggregations.com/) - Howw to develop effective and optimal data manipulation and analytics pipelines"
```
Comment (ramnes) [inline README.md:73]: I hope that you pay more attention to typos in your book. :P 

```suggestion
 - [Practical MongoDB Aggregations](https://www.practical-mongodb-aggregations.com/) - How to develop effective and optimal data manipulation and analytics pipelines
```

Also, entries in the list should be ordered alphabetically.

**Review by ramnes — CHANGES_REQUESTED** [review]

**Review by pkdone — COMMENTED** [review]
Fixed (hopefully)

**Comment by pkdone** [comment]
Fixed - hopefully

**Comment by ramnes** [comment]
Thanks! Looks like a great book, good work. :)

**Pull request merged**